api/nakadi-event-bus-api.yaml

```diff
@@ -1191,11 +1191,6 @@ definitions:
       - $ref: '#/definitions/Cursor'
       - type: object
         properties:
-          id:
-            type: string
-            format: uuid
-            description: |
-              Id of the cursor (a UUID).
           event_type_name:
```

Comment:
I have another complaint. Can this be renamed to event_type? I think it's pretty obvious that it's a name.
And I think in few other places we use event_type without a fear that people will not understand that it's a name.